public/content/en/basics/alias-strings.md

```diff
@@ -12,11 +12,11 @@ its content will never change again. And actually this is the second
 introduction: welcome UTF-8 `string`!
```

Comment:
Maybe not the right place to raise the question (I'm just browsing through my GH mails), but I don't think we need to be pedantic about this, as this is not the actual documentation.

The most important IMO is to capture the users attention, by whatever means necessary.

Many educational sites use "we". Probably the most well-known is khan academy (at least in all videos).

Also don't be afraid to step out of the box. For example,  https://rubymonk.com uses the master and apprentice metaphor to a great effect. I.e. it doesn't advertise the language but challenges the beginner/student/apprentice to become a master.

It might not be intuitive, but if we put "Hardest course ever. Only 1% successful complete it! Don't try this at home! Only for pros", etc, etc , we may actually get more interested users :D